forge-gui/res/cardsfolder/upcoming/join_the_dead.txt

```diff
@@ -0,0 +1,8 @@
+Name:Join the Dead
+ManaCost:1 B B
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | NumAtt$ -5 | NumDef$ -5 | IsCurse$ True | ConditionCheckSVar$ X | ConditionSVarCompare$ LT4 | SubAbility$ PumpDescended | SpellDescription$ Target creature gets -5/-5 until end of turn. Descend 4 — That creature gets -10/-10 until end of turn instead if there are four or more permanent cards in your graveyard.
+SVar:PumpDescended:DB$ Pump | Defined$ Targeted | NumAtt$ -10 | NumDef$ -10 | ConditionCheckSVar$ X | ConditionSVarCompare$ GE4
```

Comment:
I think Compare SVar will be a bit cleaner

Reply:
yeah it would be cleaner as one Pump effect with Variable values